core/src/test/java/fr/sncf/osrd/train/TestTrains.java

```diff
@@ -3,52 +3,55 @@
 import fr.sncf.osrd.railjson.schema.rollingstock.RJSLoadingGaugeType;
 import fr.sncf.osrd.railjson.schema.rollingstock.RJSRollingStock;
 import java.util.ArrayList;
-import java.util.Set;
 import java.util.*;
 
 public class TestTrains {
-    public static final RollingStock FAST_NO_FRICTION_TRAIN = new RollingStock(
-            "no friction train",
-            200, 1, 1, 0,
-            0,
-            0,
-            300,
-            0,
-            1,
-            1,
-            1,
-            RJSRollingStock.GammaType.CONST,
-            RJSLoadingGaugeType.G1,
-            Map.of("thermal", new RollingStock.ModeEffortCurves(
-                    false,
-                    new RollingStock.TractiveEffortPoint[] { new RollingStock.TractiveEffortPoint(0, 1) },
-                    new RollingStock.ConditionalEffortCurve[0]
-                    )
-            ),
-            "thermal"
-        );
-
     public static final RollingStock REALISTIC_FAST_TRAIN;
     public static final RollingStock REALISTIC_FAST_TRAIN_MAX_DEC_TYPE;
     public static final RollingStock VERY_SHORT_FAST_TRAIN;
     public static final RollingStock VERY_LONG_FAST_TRAIN;
     public static final RollingStock FAST_TRAIN_LARGE_GAUGE;
     public static final RollingStock FAST_ELECTRIC_TRAIN;
 
+    public static final RollingStock CONSTANT_POWER_TRAIN;
 
-    static {
-        double trainMass = 850000; // in kilos
-        double maxSpeed = 300 / 3.6;
-        var tractiveEffortCurve = new ArrayList<RollingStock.TractiveEffortPoint>();
+    private enum CurveShape {
+        LINEAR,
+        HYPERBOLIC
+    }
```

Comment:
I'm pretty confused as to why this change was introduced in this pull request, can you elaborate and add a source comment explaining what's the difference between linear and hyperbolic curves, and what it's used for?

Reply:
Alright we'll add some comments.